sdk/examples/how_tos/accounts_and_addresses/list_transactions.rs

```diff
@@ -22,10 +22,9 @@ async fn main() -> Result<()> {
         .with_storage_path(&std::env::var("WALLET_DB_PATH").unwrap())
         .finish()
         .await?;
-    let account = wallet.get_account("Alice").await?;
 
     // Sync account
```

Comment:
Update and look in the whole repo for occurrences of this

Reply:
hopefully found all of them